### PR TITLE
chore: bump frontend maven plugin version (14.8)

### PR DESCRIPTION
**Pull request by sissbruecker**
## Description

Bumps the version of `frontend-maven-plugin` that is used in the `vaadin-date-picker-flow` build to be compatible with M1 Macs. The current version of the plugin tries to download the node releases that do not exist, which results in a 404:
```
[INFO] Downloading https://nodejs.org/dist/v14.17.4/node-v14.17.4-darwin-arm64.tar.gz to /Users/sascha/.m2/repository/com/github/eirslett/node/14.17.4/node-14.17.4-darwin-arm64.tar.gz
```

Reference: https://github.com/eirslett/frontend-maven-plugin/issues/952

### CODE REVIEW

File: vaadin-date-picker-flow-parent/vaadin-date-picker-flow/pom.xml [hunk 1]
```diff
@@ -157,7 +157,7 @@
             <plugin>
                 <groupId>com.github.eirslett</groupId>
                 <artifactId>frontend-maven-plugin</artifactId>
-                <version>1.9.1</version>
+                <version>1.12.1</version>
```
Comment (sissbruecker) [inline vaadin-date-picker-flow-parent/vaadin-date-picker-flow/pom.xml:160]:
```suggestion
                <version>1.11.3</version>
```

**Comment by sissbruecker** [comment]
Updated to use `1.11.3` instead, since `1.12.x` requires Maven `3.6.x`, which we apparently don't use

**Comment by vursen** [comment]
> that is used in the vaadin-date-picker-flow build

How could I test it? Is it about compiling Flow components?

**Comment by vursen** [comment]
Confirmed that the PR fixes the error. Approved.

**Review by vursen — APPROVED** [review]

**Pull request merged**